### ISSUE TITLE
Native image for REST API Service native image

### ISSUE THREAD

**Issue by jeffreyscarpenter** [stargate/stargate#2155]
Support the option to build the REST API service as a native image (GraalVM) in Docker, since this is a supported option with Quarkus framework.

**Comment by jeffreyscarpenter** [comment]
May be dependent on #1599?

**Comment by jeffreyscarpenter** [comment]
Won't do
